src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsKeyValuePairValidator.java

```diff
@@ -36,6 +40,8 @@ public CaseResponse validate(List<OcrDataField> ocrDataValidationRequest, FormTy
         try {
             if (formType != null && formType.equals(FormType.SSCS2)) {
                 sscs2Schema.validate(new JSONObject(build(ocrDataValidationRequest)));
+            } else if (formType != null && formType.equals(FormType.SSCS5)) {
+                sscs5Schema.validate(new JSONObject(build(ocrDataValidationRequest)));
```

Comment:
why not load the data and then validate here it rather than copy the if else statement?

Reply:
Updated